### PR TITLE
Add a document listing all APIcast parameters

### PR DESCRIPTION
**Pull request by mayorova**
Meant to be a verbose document (in addition to `.env` and `apicast-template.yml`) listing all parameters that can be used to modify the APIcast v2 behavior.

There are two links to documents that do not exist yet, but are planned to be added: `/doc/custom-config.md` and `/doc/path-routing.md`

### CODE REVIEW

**Comment by mikz** [comment]
This might be good opportunity to investigate and define https://github.com/3scale/apicast/issues/207

**Comment by mayorova** [comment]
Not sure how to proceed with this one btw.
Should I remove the non-existing links, until the referenced articles are created, and merge this doc?

**Comment by mikz** [comment]
@mayorova good enough for me! If you keep it as a separate commit it can be reverted later.

File: doc/parameters.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+# APIcast parameters
+
+APIcast v2 has a number of parameters configured as environment variables that can modify the behavior of the gateway. The following reference provides descriptions of these parameters.
+
+Note that when deploying APIcast v2 with OpenShift, some of thee parameters can be configured via OpenShift template parameters. The latter can be consulted directly in the [template](https://raw.githubusercontent.com/3scale/apicast/v2/openshift/apicast-template.yml).
```
Comment (mikz) [inline doc/parameters.md]: those `/v2` links don't work anymore

Comment (mayorova) [inline doc/parameters.md]: Argh, true.

**Comment by mayorova** [comment]
Probably mass replace would be good.

**Comment by mikz** [comment]
I definitely want to add a link checker to the CI build before doing the mass rename so I can verify new links work.

**Pull request merged**